Start stop node

elraphty opened the pull request:
This feature changes the text color, and border color of an exited node, using events and svelte derivations
@Evanfeenstra

Evanfeenstra commented:
i fixed a bunch of bugs when clicking into node controls when they are are stopped. But there are probably more JS errors @elraphty

elraphty commented:
> i fixed a bunch of bugs when clicking into node controls when they are are stopped. But there are probably more JS errors @elraphty

Thanks @Evanfeenstra I fixed some undefined errors using the Javascript coalescing operator (??), I will be more proactive on such bugs going forward.

Evanfeenstra commented:
@elraphty i dont see another commit to this PR...
